api/build.gradle

```diff
@@ -351,8 +350,8 @@ project.tasks.ideaModule.dependsOn(generateSwaggerCode)
 // Swagger sources, per the swaggerSources tuple above.
 project.tasks.compileGeneratedJava.dependsOn(generateSwaggerCode)
 
-project.tasks.bootWar.dependsOn(['compileGeneratedJava', 'generate_appengine_web_xml',
-                             'generate_appengine_cron_yaml'])
+project.tasks.bootJar.dependsOn(['compileGeneratedJava', 'generate_appengine_yaml',
```

Comment:
I noticed this changed from `bootWar` to `bootJar`. Does that mean we can remove `apply plugin: 'war'` ?